### ISSUE TITLE
joss paper comments

### ISSUE THREAD

**Issue by slwu89** [arcaldwell49/SimplyAgree#11]
I found this sentence hard to parse:

> Based on citation counts, researchers highly value methods to quantify the accuracy and reliability of the measurement tools

Does this mean that methods that quantify the accuracy and reliability of tools are highly cited? Or that tools that have been shown to be accurate and reliable are highly cited? I'd suggest revising for clarity.

Another comment involves indicating more clearly who can benefit from the functionality in this package. For example, can this aid persons wishing to compare completely arbitrary tools that return quantitative (numeric) measurements of some arbitrary process? Are categorical or discrete measurements allowed?

Final comment: the review guidelines specify:

> State of the field: Do the authors describe how this software compares to other commonly-used packages? 

The paper does remark that open source software to support these analyses is lacking. If there are any existing R packages however which implement some of these methods, it would be good to cite them and briefly say why they are inadequate, or if none, clearly state that the R ecosystem currently has no such software.

**Comment by arcaldwell49** [comment]
Great points.

1. I have modified the opening 2 sections to address the first point. 

> Based on citation counts of @bland1986 and @weir2005 alone^[@bland1986 and @weir2005 have greater than 50000 and 4000 citations, respectively, at the time of writing this manuscript. Many of these publications are original research investigations.], researchers appear to highly value methods to quantify the accuracy and reliability of measurement tools used for research.

I have also modified the summary section to clarify that SimplyAgree only supports continuous measurements at this time.

2. I have added to the "statement of need" section to specifically state the known packages that exist for providing similar calculations and what differentiates SimplyAgree from those packages.

> While some software may provide the limits of agreement analysis outlined by Bland & Altman [-@bland1986; -@bland1999], few, if any, account for multiple observations within the same research subject [@zou2011] or include hypothesis tests of agreement [@shieh2019]. `blandr` [@blandr] and `BlandAltmanLeh` [@BlandAltmanLeh] exist to aid the creation of Bland-Altman type plots, but do not offer the tests detailed by @shieh2019 or @zou2011. `MethComp` [@MethComp] is the most comprehensive with regards to limits of agreement but also lacks the methods outlined by @shieh2019. The `cccrm` package [@carrasco2013] does already provide concordance correlation coefficients as output, and is the basis of the concordance calculations within `SimplyAgree`. Lastly, `psych` [@psych] and `ICC` [@ICC] provide some measures of reliability but lack the ability to calculate standard error of the measurement. Moreover, only `blandr` has a jamovi module but the capabilities are limited to just the traditional Bland-Altman limits.

All changes should be viewable in the main branch in the paper.md file

**Comment by icc** [comment]
What?
